src/components/itemsCardsList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ItemCard from './itemCard';
+import { IItemData } from '../data/items';
+import './itemsCardsList.css';
+
+interface IItemsCardsListProps {
+  heightOffset: string;
+  baseImgUrl: string;
+  items: IItemData[];
+}
+
+function ItemsCardsList({ heightOffset, baseImgUrl, items }: IItemsCardsListProps) {
+  return (
+    <section
```

Comment:
I would move `section` out of `ItemsCardsList`
Pros:
1) You can reuse it in different places on the site since there is no section limitations. (In the futute you might want to render it inside of a div, article and so on)
2) You don't have to pass `heightOffset` prop